app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+    return sum(teammate.get_rating() for teammate in team)
+
+
+def elves_concert(band: list[Elf]) -> None:
```

Comment:
Consider using a more descriptive parameter name, such as `elves` instead of `band`